respec/Rationale&usecases.md

```diff
@@ -1,45 +1,46 @@
+Het domein voor het voorliggende profiel voor een begrippencatalogus is 1 op 1 het domein zoals beschreven op de pas-toe-leg-uit-lijst. Praktisch betekent dit dat dit profiel primair een profiel is voor de toepassing van SKOS. In het kader van 'practice what you preach' worden alle gebruikte onderdelen in dit profiel beschreven in SKOS, toegepast volgens dit profiel. Alle definities van de onderdelen zijn 1 op 1 overgenomen uit de SKOS primer. Door de onderdelen als SKOS concept te beschrijven is het eenvoudig ook een Nederlandse vertaling van deze definitie op te nemen en scope notities (toelichtingen) toe te voegen die de verbinding leggen met de ISO 25964. Daarnaast wordt via [dct:source](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/source) steeds expliciet de relatie gelegd naar de bron(nen) waaraan de beschrijvingen in dit profiel zijn ontleend.
 ## Use cases
 Het profiel is modulair opgebouwd. Hieronder beschrijven we 4 uses cases die overeenkomen met verschillende niveaus van diepgang waarop concepten in samenhang kunnen worden beschreven. Iedere use case wordt ondersteund door een bouwsteen in het profiel Door nieuwe bouwstenen toe te voegen kan steeds meer semantiek aansamenhangende concepten worden toegevoegd zodat een taxonomie, een thesaurus ontstaat of zelfs een thesaurus inclusief annotatie van voor rechtshandelingen relevante concepten ontstaat.
 
-De 4 uses cases zijn: 
-1. **Glossary**. Dit is de meest eenvoudige use case, die een baseline geeft die voor iedereen  geldt. In het geo domein is het vaak voldoende om de dingen die op de kaart moeten komen als concept te definiëren, zonder verdere onderlinge relaties tussen de concepten te definiëren. Op dit niveau zijn de volgende user stories gedefinieerd:
+De 4 uses cases zijn:
+1. **Glossary**. Dit is de meest eenvoudige use case, die een baseline geeft die voor iedereen  geldt. In het geo domein is het vaak voldoende om de dingen die op de kaart moeten komen als begrip te definiëren, zonder verdere onderlinge relaties tussen de begrippen te definiëren. Op dit niveau zijn de volgende user stories gedefinieerd:
```

Comment:
Kan iemand uit het geo-domein bevestigen wat hier berweerd wordt?

> In het geo domein is het vaak voldoende om de dingen die op de kaart moeten komen als begrip te definiëren, zonder verdere onderlinge relaties tussen de begrippen te definiëren.

Suggestie: 

> In het geo-domein wordt vaak volstaan met de dingen die op de kaart moeten komen als begrip te definiëren, zonder verdere onderlinge relaties tussen de begrippen te definiëren.

Maar dat is net zo goed een praktijk in de meeste domeinen!

> Vaak wordt volstaan met het definiëren van begrippen, zonder verdere onderlinge relaties tussen de begrippen te leggen.

Reply:
Aangepast naar meer generieke niet-specifiek geo omschrijving, zoals voorgesteld.